SharedPluginFiles/index.html

```diff
@@ -3,5 +3,8 @@
 </head>
 <body>
 <h1>Shared Plugin Files</h1>
+<p>
+    This is a blank HTML, which seems to be required in order to serve images from this directory.
```

Comment:
I just put this note in here for future reference. I can show you this behavior to you if you're curious.

Reply:
Let's look next week if we remember.